jira_docugen.tex

```diff
@@ -0,0 +1,973 @@
+\subsection{Test Cases Summary}\label{test-cases-summary}
+
+\begin{longtable}[]{p{3cm}p{13cm}}
+\toprule
+Jira Id & Test Name\tabularnewline
+\midrule
+\endhead
+\protect\hyperlink{lvv-t17---ag-00-00-installation-of-the-alert-generation-science-payload}{LVV-T17}
+&
+\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T17}{AG-00-00:
+Installation of the Alert Generation science payload.}\tabularnewline
+\protect\hyperlink{lvv-t18---ag-00-05-alert-generation-produces-required-data-products}{LVV-T18}
+&
+\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T18}{AG-00-05:
+Alert Generation Produces Required Data Products}\tabularnewline
+\protect\hyperlink{lvv-t19---ag-00-10-scientific-verification-of-processed-visit-images}{LVV-T19}
+&
+\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T19}{AG-00-10:
+Scientific Verification of Processed Visit Images}\tabularnewline
+\protect\hyperlink{lvv-t20---ag-00-15-scientific-verification-of-difference-images}{LVV-T20}
+&
+\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T20}{AG-00-15:
+Scientific Verification of Difference Images}\tabularnewline
+\protect\hyperlink{lvv-t21---ag-00-20-scientific-verification-of-diasource-catalog}{LVV-T21}
+&
+\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T21}{AG-00-20:
+Scientific Verification of DIASource Catalog}\tabularnewline
+\protect\hyperlink{lvv-t22---ag-00-25-scientific-verification-of-diaobject-catalog}{LVV-T22}
+&
+\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T22}{AG-00-25:
```

Comment:
There is a problem here.  You only show one test specification (AG-00, small scale alert generation), but the test cases also include the AD-XX-YY alert distribution tests, which should be separate.

Reply:
Since the old naming is not valid anymore, we agreed to move the specification design information up in section 2. Section 3 will remain just a list of test cases included in the test specification.